internal/dataplane/sendconfig/dbmode.go

```diff
@@ -59,12 +71,13 @@ func (s UpdateStrategyDBMode) Update(ctx context.Context, targetContent ContentW
 		TargetState:     ts,
 		KongClient:      s.client,
 		SilenceWarnings: true,
+		IsKonnect:       s.isKonnect,
 	})
 	if err != nil {
 		return fmt.Errorf("creating a new syncer for %s: %w", s.client.BaseRootURL(), err), nil, nil
 	}
 
-	_, errs := syncer.Solve(ctx, s.concurrency, false)
```

Comment:
Apparently, this `deck` release made a breaking change in the syncer API. It extends `Solve` method with a parameter `isJSONOut` which when turned `true`, returns a non-empty `EntityChanges` struct as a third return value (it includes stats about objects CRUD actions). I set it to false as we won't use it.